src/Servers/Kestrel/Core/src/Internal/Http3/Http3Stream.cs

```diff
@@ -1169,7 +1169,9 @@ private Pipe CreateRequestBodyPipe(uint windowSize)
         }
     }
 
+#pragma warning disable CA2252 // WebTransport is a preview feature
     public override async ValueTask<IWebTransportSession> AcceptAsync(CancellationToken token)
```

Comment:
Should this API have the experimental attribute?

Reply:
This class is internal, it shouldn't matter.